packages/repo-tools/src/commands/openapi/client/generate.ts

```diff
@@ -74,8 +67,9 @@ async function generate(spec: string, outputDirectory: string) {
     `yarn backstage-cli package lint --fix ${resolvedOutputDirectory}`,
   );
 
-  if (cliPaths.resolveTargetRoot('node_modules/.bin/prettier')) {
-    await exec(`yarn prettier --write ${resolvedOutputDirectory}`);
+  const prettier = cliPaths.resolveTargetRoot('node_modules/.bin/prettier');
+  if (prettier) {
+    await exec(`${prettier} --write ${resolvedOutputDirectory}`);
```

Comment:
I don't get this one either. `yarn` will look in `node_modules/.bin`. Just plain executing the script name won't. This change seems to me like it actually broke the previous functionality, at a glance, such that now, prettier actually has to be installed globally by the user in their environment and have it on their $PATH, rather than being able to leverage it as depended on by the monorepo.

If we are able to do `yarn backstage-cli` a few lines above (which uses the exact same mechanics), why not `yarn prettier` here?

You mention in the PR comment that the previous code required prettier to be installed in the monorepo, but that seems to me to be the desired behavior: if it's installed, we detect that and use it, but if it isn't, we trust that you don't want to use prettier at all.